src/index/am.rs

```diff
@@ -199,6 +203,33 @@ pub unsafe extern "C" fn ambuild(
             (*state.result).heap_tuples += 1.0;
         }
     }
+    let mut rpc = builder.rpc;
+    match rpc.start(handle) {
+        Ok(()) => (),
+        Err(StartError::NotExist) => pgrx::error!("internal error"),
+    }
+    loop {
+        pgrx::check_for_interrupts!();
+        match rpc.stat(handle) {
+            Ok(s) => {
+                if !s.indexing {
+                    break;
+                }
+            }
+            Err(StatError::NotExist) => pgrx::error!("internal error"),
+        }
+        unsafe {
```

Comment:
Can you add some comments about `WaitLatch`?

Reply:
No. I just copy the code from `pg_sleep`.